app/assets/stylesheets/summary-card.scss

```diff
@@ -0,0 +1,3 @@
+.bold-border-top {
+  border-top: 2px solid #000000
```

Comment:
Design system: should we use the GDS Sass variables or colour palette instead of the hex value?